var/spack/repos/builtin/packages/brahma/package.py

```diff
@@ -16,10 +16,13 @@ class Brahma(CMakePackage):
     version("develop", branch="dev")
     version("master", branch="master")
     version("0.0.1", tag="v0.0.1", commit="15156036f14e36511dfc3f3751dc953540526a2b")
+    version("0.0.2", tag="v0.0.2", commit="1bac58d5aa8962a5c902d401fbf8021aff9104d3")
```

Comment:
Is this the correct commit?  It looks like the commit starts: `bac58d5`, not with a `1`.

Reply:
Fixed.